frontend/uddhar-frontend/src/public/components/HeroSection.jsx

```
@@ -0,0 +1,54 @@
+    return () => clearInterval(interval);
+  }, []);
+
+  return (
+    <div className="flex items-center justify-around h-screen max-md:h-auto bg-gradient-to-b from-amber-200 to-gray-100 px-5 max-md:flex-col">
+      {/* Left: Image Stack */}
+      <div className="relative max-md:w-full w-1/2 min-md:min-h-100 h-130 overflow-hidden max-md:min-w-100 mx-10">
+        {heroImg.map((img, i) => (
+          <motion.img
+            key={i}
+            src={img}
+            alt={`Image ${i}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)